wks-recruiter-interface/src/app/layout/tests/tests-create/tests-create.component.ts

```diff
@@ -17,14 +17,24 @@ export class TestsCreateComponent implements OnInit {
   public test: Test;
   private title: String;
   private modifying: Boolean;
+  private languages;
 
   constructor(public activeModal: NgbActiveModal,
     private alertsService: AlertsService,
     private router: Router,
     private testsService: TestsService) { }
 
   ngOnInit() {
-
+    this.languages = [
+      {id: "polish", name: "polish"},
```

Comment:
tak samo jak wyżej, skoro id jest takie samo jako name to nie można zrobić tablicy stringów